src/pages/DataLoader/styles.scss

```diff
@@ -0,0 +1,90 @@
+$width: 400px;
+
+.data-loader {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Can you please align it left and make the width wider, it's very cramped in the center of my screen